doc/git-guidelines.md

```diff
@@ -1,105 +1,134 @@
-The git guidelines for SteemIt are influenced by the
-[Graphene](https://github.com/cryptonomex/graphene/wiki/How-we-use-version-control)
-git guidelines as well as [Git
-Flow](http://nvie.com/posts/a-successful-git-branching-model/) and [this
-blog
-post](http://www.draconianoverlord.com/2013/09/07/no-cherry-picking.html).
-
 ## Branches
-- `stable`: Points to the latest version of code that is production-ready, and has been tested in production.
- Witnesses, exchanges, seed nodes, and RPC nodes should be running this branch. 
-- `master`: The active development branch. We will strive to keep `develop`
-  in a working state. All PRs must pass automated tests before being merged.
-  While this is not a guarantee that `master` is bug-free, it will
-  guarantee that the branch is buildable in our standard build configuration
-  and passes the current suite of tests. That being said, running a node
-  from `master` has risks.  We recommend that any block producing node
-  build from `stable`. If you want to test new features, develop is the
-  correct branch.
+- `stable`: Points to the latest version of code that is production-ready, and has been tested in production. This is the branch we recommend for exchanges.
+- `0.21.x`: This is the current release branch and will always point to the most recent release. Witnesses and seed node operators may want to run this branch. `stable` and `0.21.x` may diverge if there are minor bug fixes or witness only security fixes that does not need exchanges to update for. 
+- `master`: The active development branch. We will strive to keep `develop` in a working state. All PRs must pass automated tests before being merged. While this is not a guarantee that `master` is bug-free, it will guarantee that the branch is buildable in our standard build configuration and passes the current suite of tests. That being said, running a node from `master` has risks.  We recommend that any block producing node build from `stable`. If you want to test new features, develop is the correct branch.
```

Comment:
Should be, ... "We will strive to keep `master` in a working state."

Reply:
> We recommend that any block producing node build from `stable`. If you want to test new features, develop is the correct branch.

Probably should be:

We recommend that any block producing node build from the latest tag. If you want to test new features, `master` is the correct branch.